learn/cookbooks/docker.md

```diff
@@ -0,0 +1,135 @@
+# Using Meilisearch with Docker
+
+A common way of installing Meilisearch is using Docker. In this guide you will learn what Docker is, how to install it in your local machine, and how to use Meilisearch with it.
+
+## Docker
+
+Docker is a tool that packages an application's code and dependencies into something called a container. Using Docker allows developers to ensure their software runs reliably across a wide range of machines and environments.
+
+Docker containers are distributed in images. Docker images are packages storing not only the container, but also instructions on how Docker should instantiate that container.
+
+## Install Docker
+
+The first step to start using Docker is installing it in your target machine. We recommend following [Docker's official instructions to install Docker Desktop](https://docs.docker.com/get-docker/), available for MacOS, Windows, and Linux.
+
+Docker for Desktop includes an interface for managing your containers, command-line applications and several tools required for using Docker.
+
+## Install Meilisearch
+
+Once you installed Docker you can use the `docker pull` command to download a Meilisearch image:
+
+```sh
+docker pull getmeili/meilisearch:v0.28
+```
```

Comment:
Like @irevoire I don't think the pull order is really necessary, but on the other hand, I love the explanations on the version.
Also, I find the title a bit confusing because we are not going to `install Meilisearch`, maybe something with an `image` could be more accurate. `Use Meilisearch image`?

Reply:
>  I love the explanations on the version

Yes, if we remove the `pull` step, we should totally keep the explanation and move it to the `run` step.

Reply:
I'll update the title—you're right, "Install" is not appropriate when talking about Docker images and can lead to confusion.